04LessonFour/in-class-project/index.html

```diff
@@ -2,9 +2,12 @@
 <html>
 <head>
   <title>The Daily Planet</title>
-  <link rel="stylesheet" href="css/style.css">
+  <!-- <link rel="stylesheet" type="text/css" href="css/style.css"> -->
+
 </head>
 <body>
-
+  <div style="color :blue;

```

Comment:
In CSS, separate style declarations would go on individual lines for clarity, but try not to do this in HTML; one single tag shouldn't span across multiple lines